content/docs/iac/concepts/resources/components.md

```diff
@@ -142,48 +142,48 @@ This example demonstrates both the naming convention and how to designate the co
 {{% choosable language javascript %}}
 
 ```javascript
-let bucket = new aws.s3.Bucket(`${name}-bucket`,
+let bucket = new aws.s3.BucketV2(`${name}-bucket`,
     {/*...*/}, { parent: this });
 ```
 
 {{% /choosable %}}
 {{% choosable language typescript %}}
 
 ```typescript
-let bucket = new aws.s3.Bucket(`${name}-bucket`,
+let bucket = new aws.s3.BucketV2(`${name}-bucket`,
     {/*...*/}, { parent: this });
 ```
 
 {{% /choosable %}}
 {{% choosable language python %}}
 
 ```python
-bucket = s3.Bucket(f"{name}-bucket",
+bucket = s3.BucketV2(f"{name}-bucket",
```

Comment:
not sure why there is an "f" here.

Reply:
This is a Python feature called f-strings https://docs.python.org/3/tutorial/inputoutput.html#formatted-string-literals - I think this is quite idiomatic these days.

Reply:
ahh thank you!!